internal/builder/packages.go

```diff
@@ -180,7 +180,7 @@ func BuildPackage(options BuildOptions) (string, error) {
 		return "", errors.Wrap(err, "resolving external fields failed")
 	}
 
-	logger.Debug("Add dynamic mappings if needed")
+	logger.Debug("Add dynamic mappings if needed (technical preview)")
```

Comment:
Would it be interesting to add this ? Maybe changing to INFO instead ?

Reply:
Sounds good to add this. If changed to info it should probably appear only if the feature is used.

Reply:
Ok! I'll move this message as an INFO to be shown just when the feature is enabled/used.